docs/howto/openstack/nova/move-instance-between-places.md

```diff
@@ -0,0 +1,119 @@
+# Move an instance from one region to another
+
+This guide will show you how to move or copy your instance volume to a different region in {{extra.brand}} Cloud.
+
+## Prerequisites
+
+- Installed OpenStack [CLI client](../../../getting-started/enable-openstack-cli/)
+- Access to source and target location (RC-file)
+- Enough space on the local machine that you download your server to
+- Target location configured with Network, Security Groups etc
+
+## Recommendations
+
+Use the UUID of the instance instead of using the instance name, this will make sure you are using the correct instance.
```

Comment:
Use "server", not "instance".

Also, the second part (after the comma) is a full sentence with a subject, verb, and object. That means that you should convert the comma into a semicolon, or better still, split the sentence there and make it two.